chia-protocol/src/classgroup.rs

```diff
@@ -7,18 +7,42 @@ use pyo3::prelude::*;
 
 streamable_struct!(ClassgroupElement { data: Bytes100 });
 
-#[cfg(feature = "py-bindings")]
-#[cfg_attr(feature = "py-bindings", pymethods)]
 impl ClassgroupElement {
-    #[staticmethod]
     pub fn get_default_element() -> ClassgroupElement {
         let mut data = [0_u8; 100];
         data[0] = 0x08;
         ClassgroupElement { data: data.into() }
     }
 
-    #[staticmethod]
-    pub fn get_size(_constants: pyo3::PyObject) -> i32 {
+    pub fn get_size() -> i32 {
         100
     }
 }
+
+#[cfg(feature = "py-bindings")]
+#[pymethods]
+impl ClassgroupElement {
+    #[staticmethod]
+    pub fn create(bytes: &[u8]) -> ClassgroupElement {
+        if bytes.len() == 100 {
+            ClassgroupElement { data: bytes.into() }
+        } else {
+            assert!(bytes.len() < 100);
+            let mut data = [0_u8; 100];
+            data[..bytes.len()].copy_from_slice(bytes);
+            ClassgroupElement { data: data.into() }
+        }
+    }
+
+    #[staticmethod]
+    #[pyo3(name = "get_default_element")]
+    pub fn py_get_default_element() -> ClassgroupElement {
+        Self::get_default_element()
+    }
+
+    #[staticmethod]
+    #[pyo3(name = "get_size")]
+    pub fn py_get_size() -> i32 {
+        Self::get_size()
```

Comment:
the reason for having the underlying rust functions be separate is that the PyO3-attributes (e.g. `#[staticmethod]`) won't work if the python bindings aren't enabled. They aren't enabled for `cargo clippy` for instance. Separating them like this, and disabling the whole `impl ClassgroupElement` when the python bindings are disabled, makes it work.